slam-libraries/viam-cartographer/src/slam_service/config_test.cc

```diff
@@ -334,6 +334,56 @@ BOOST_AUTO_TEST_CASE(ParseAndValidateConfigParams_valid_config_no_camera) {
     delete argv;
 }
 
+BOOST_AUTO_TEST_CASE(
```

Comment:
can incorporate into other tests if needed, but felt it nice to break out.

Reply:
See orbslam tests for example of including in current BOOST tests

Reply:
I think breaking this out is better and should be done in the same way in the orbslam tests.